docs/plans/2026-04-28-001-feat-editor-asset-library-plan.md

```
@@ -0,0 +1,856 @@
+}, [activeTeam?.id]);
+```
+
+Why this differs from v1 of the plan:
+- **`cache-first`, not `cache-and-network`.** With single-source-of-truth mutation eviction, every drawer open is either a cache hit (fresh) or a fresh fetch (just evicted). `cache-and-network` would double-pay every open.
+- **`relayStylePagination(['scope'])` as the field policy** — handles edge merging, cursor stability, dedup-by-cursor automatically. Source: https://www.apollographql.com/docs/react/pagination/cursor-based#relay-style-cursor-pagination
+- **`ImageBlock.keyFields: false`** prevents the library's `ImageBlock` nodes from sharing cache identity with the editor's canonical `ImageBlock` queries (where they would otherwise overwrite richer field selections).
+- **Concurrent `fetchMore` guard** via `notifyOnNetworkStatusChange: true` + `NetworkStatus.fetchMore` (R14).
+
+#### Frontend file layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)